images/rabbitmq/Dockerfile

```diff
@@ -3,9 +3,20 @@ FROM rabbitmq:3-management
 ARG LAGOON_VERSION
 ENV LAGOON_VERSION=$LAGOON_VERSION
 
-COPY rabbitmq_delayed_message_exchange-3.7.0.ez /plugins
+ENV RABBITMQ_DEFAULT_USER=guest \
+    RABBITMQ_DEFAULT_PASS="guest"\
+    RABBITMQ_DEFAULT_HA_PATTERN='^lagoon-'
```

Comment:
let's use as default `^$`, aka a pattern that does not match.

Reply:
I need to test the behavior of such pattern, because it could match ALL queues, either temporary queues.

Reply:
Tested and the pattern is not matched by default by any queue